spec/views/text_components/index.html.haml_spec.rb

```diff
@@ -38,6 +50,14 @@
     expect(rendered).to include('Events')
   end
 
+  it "contains dropdown menus for sensors and events to insert markup into textareas" do
+    render
+    parsed = Capybara.string(rendered)
+    parsed.all('form').each do |form|
+      expect(form).to have_selector('.text-editor__toolbar__item.dropdown-toggle', :count => 2)
+    end
+  end
+
```

Comment:
As far as I can tell, there is one rspec test that effectively prevents sensors from being rendered, if they are not connected to a TextComponent via a Trigger. The behaviour should be the exact opposite now.